public/directives/wz-logtest/components/logtest.tsx

```diff
@@ -81,7 +81,19 @@ export const Logtest = compose(
     );
   };
 
+  const getTokenSessionLogtest = async () => {
+    const body = {
+      log_format: 'syslog',
+      location: 'logtest',
+      event: 'get token logtest session',
+    };
+    return await WzRequest.apiReq('PUT', '/logtest', body)
+    .then(res => res.data.data.token)
+    .catch(error => '') 
```

Comment:
This code suggests to me that it is doing a request with a fake log to get the token. This should avoid.

Could we get the token with the request with the first log instead, and use it in the next request if there are more logs?

Reply:
Exactly! 👍🏼